### PR TITLE
Add support to abort all or a specific response

### PR DESCRIPTION
**Pull request by nemanja-boric-sociomantic**

### CODE REVIEW

File: src/swarm/neo/client/mixins/ClientCore.d [hunk 1]
```diff
@@ -857,7 +855,24 @@ template ClientCore ( )
 
     public void abortAllRequests ( )
     {
-        // TODO
+        this.connections.request_set.abortAllRequests();
+    }
+
+    /***************************************************************************
+
+        Aborts a specific requests. This is useful if there's no way to
```
Comment (gavin-norman-sociomantic) [inline src/swarm/neo/client/mixins/ClientCore.d:863]: Hmm, I'd be inclined to not support this until we have a really good need for it.

File: src/swarm/neo/client/mixins/ClientCore.d [hunk 2]
```diff
@@ -857,7 +855,24 @@ template ClientCore ( )
 
     public void abortAllRequests ( )
     {
-        // TODO
+        this.connections.request_set.abortAllRequests();
```
Comment (gavin-norman-sociomantic) [inline src/swarm/neo/client/mixins/ClientCore.d:858]: Bearing in mind the comment below, should this method also reset all the connections?

**Comment by gavin-norman-sociomantic** [comment]
Preparing to release v4.7.0. Pushing this to v4.8.0, as it's not urgently needed.

File: src/swarm/neo/client/mixins/ClientCore.d (hunk 1 above)
Comment (gavin-norman-sociomantic) [reply src/swarm/neo/client/mixins/ClientCore.d:863]: Maybe should we close this PR?

Comment (nemanja-boric-sociomantic) [reply src/swarm/neo/client/mixins/ClientCore.d:863]: Yes, it's a mistake that it's still open

**Pull request closed without merging**